internal/test/tx.go

```diff
@@ -0,0 +1,11 @@
+package test
```

Comment:
Now that you have brought back the `internal` directory, I think it'd make sense to delete `test/factory/tx.go` in favor of this file (they are duplicated AFAICT).